src/index.scss

```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```

Comment:
The feature flags mapping is needed.

Reply:
I've commited to my branch. Not sure how or where to commit to this PR.